src/jti_background.js

```diff
@@ -112,13 +144,18 @@ function setToggleStatus (toggleType, toggleInput, callback) {
 }
 
 function clearStorage (callback) {
```

Comment:
If this function is only clearing the templates. Then I think the name of the function needs to be updated.